sdk/tables/Azure.Data.Tables/src/TableInternalMergeEntityHeaders.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure;
+using Azure.Core;
+
+namespace Azure.Data.Tables
+{
+    internal class TableInternalMergeEntityHeaders
```

Comment:
Why did you have to do this?

Reply:
Because Merge is totally missing from the swagger 
https://github.com/Azure/azure-rest-api-specs/pull/8151/files/f5cb6fb416ae0a06329599db9dc17c8fdd7f95c7#r417331328